main.go

```diff
@@ -98,7 +100,8 @@ func main() {
 	}
 
 	if *scriptPath != "" {
-		script, err := ioutil.ReadFile(*scriptPath)
+		var script []byte
```

Comment:
why move this out?

Reply:
same here, vetshadow

Reply:
let's keep this is as is, the shadowing is not going to cause issues.